pom.xml

```diff
@@ -331,7 +338,35 @@
             </resource>
         </resources>
         <plugins>
-
+            <plugin>
+                <groupId>dk.kb.util</groupId>
+                <artifactId>injection-maven-plugin</artifactId>
+                <version>1.0.0</version>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>read-yaml-properties</goal>
+                        </goals>
+                    </execution>
+                </executions>
+                <configuration>
+                    <filePath>conf/ds-storage-behaviour.yaml</filePath>
+                    <yamlResolvers>
+                        <YamlResolver>
+                            <yamlType>List</yamlType>
+                            <yamlPath>allowed_origins</yamlPath>
+                            <sourceKey>name</sourceKey>
+                            <destinationKey>allowed_origins</destinationKey>
+                            <createEnum>true</createEnum>
+                        </YamlResolver>
+                        <YamlResolver>
+                            <yamlType>Single-value</yamlType>
```

Comment:
Running
```shell
mvn clean package -Dskiptests
```
gives me
```
[ERROR] Failed to execute goal dk.kb.util:injection-maven-plugin:1.0.0:read-yaml-properties (default)
  on project ds-storage: Unable to parse configuration of mojo dk.kb.util:injection-maven-plugin:1.0.0:
  read-yaml-properties for parameter sourceKey: Cannot find 'sourceKey' in class dk.kb.util.YamlResolver
 -> [Help 1]
```

Reply:
Doh. I've been to fast. Sorry!

Reply:
My guess is that you've installed the plugin manually earlier. Could you please remove the 1.0.0 version installed locally in your .m2 repo and then re-run the command?